### ISSUE TITLE
Formaliser l'affichage des messages d'erreur

### ISSUE THREAD

**Issue by htmfilho** [uclouvain/osis-documentation#54]
@glamarca commented on [Wed Apr 27 2016](https://github.com/uclouvain/osis/issues/500)

**Comment by basmot** [comment]
@glamarca @htmfilho Ce ticket a encore du sens?

**Comment by htmfilho** [comment]
Oui, parce que il manque de formalisation. Le comportement n'est pas le même par tout.
